docs/modules/ROOT/pages/api/introspection.adoc

```diff
@@ -40,45 +39,48 @@ on how to compute this ID.
 [[SRC5]]
 === `++SRC5++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/introspection/src5.cairo[{github-icon},role=heading-link]
 
-[.hljs-theme-light.nopadding]
 ```javascript
 use openzeppelin::introspection::src5::SRC5;
 ```
 
 SRC5 contract implementation extending xref:ISRC5[`ISRC5`].
 
 [.contract-index]
-.Functions
+.External Functions
 --
 [.contract-subindex-inherited]
-.InternalImpl
+.SRC5Impl
 
-* xref:#SRC5-register_interface[`++register_interface(ref self: ContractState, interface_id: felt252)++`]
-* xref:#SRC5-deregister_interface[`++deregister_interface(ref self: ContractState, interface_id: felt252)++`]
+* xref:#SRC5-supports_interface[`++supports_interface(self, interface_id)++`]
+--
 
+[.contract-index]
+.Internal Functions
+--
 [.contract-subindex-inherited]
-.SRC5Impl
+.InternalImpl
 
-* xref:#SRC5-supports_interface[`++supports_interface(self: @ContractState, interface_id: felt252)++`]
+* xref:#SRC5-register_interface[`++register_interface(self, interface_id)++`]
+* xref:#SRC5-deregister_interface[`++deregister_interface(self, interface_id)++`]
 --
 
 [#SRC5-Functions]
 ==== Functions
 
+[.contract-item]
+[[SRC5-supports_interface]]
```

Comment:
we should separate between `Internal` and `External` for consistency with the rest of the PRs